crypto/s2n_drbg.c

```diff
@@ -136,10 +136,28 @@ int s2n_drbg_instantiate(struct s2n_drbg *drbg, struct s2n_blob *personalization
     return 0;
 }
 
+int s2n_new_aes128_drbg(struct s2n_drbg *drbg, struct s2n_blob *personalization_string)
+{
+    drbg->ctx = EVP_CIPHER_CTX_new();
+    S2N_ERROR_IF(!drbg->ctx, S2N_ERR_DRBG);
+    (void)EVP_CIPHER_CTX_init(drbg->ctx);
+    GUARD_OSSL(EVP_EncryptInit_ex(drbg->ctx, EVP_aes_128_ecb(), NULL, NULL, NULL), S2N_ERR_DRBG);
+    return s2n_drbg_instantiate(drbg, personalization_string);
+}
+
+int s2n_new_aes256_drbg(struct s2n_drbg *drbg, struct s2n_blob *personalization_string)
```

Comment:
This code is basically a code duplication of the above code, except for EVP_aes_128_ecb()/EVP_aes_256_ecb(), another reason to use an enum instead of two APIs (see comment in the header file).